src/java/com/unifina/utils/ImageResizer.java

```diff
@@ -0,0 +1,48 @@
+package com.unifina.utils;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+public class ImageResizer {
+	private static final int HERO_WIDTH = 520;
+	private static final int HERO_HEIGHT = 400;
+	private static final int THUMB_WIDTH = 360;
+	private static final int THUMB_HEIGHT = 210;
+	public enum Size {
+		HERO(HERO_WIDTH, HERO_HEIGHT),
+		THUMB(THUMB_WIDTH, THUMB_HEIGHT);
+
+		private final int width;
+		private final int height;
+		Size(final int width, final int height) {
+			this.width = width;
+			this.height = height;
+		}
+		public int width() { return width; }
+		public int height() { return height; }
+	}
+
+	public byte[] resize(byte[] image, String filename, Size size) throws IOException {
+		final BufferedImage original = ImageIO.read(new ByteArrayInputStream(image));
+		final Image tmp = original.getScaledInstance(size.width(), size.height(), Image.SCALE_SMOOTH);
```

Comment:
Cropping from center would probably be better than stretching (ping @juhah), that is, something like
```
final float aspectRatio = (float)size.width() / (float)size.height();
final int w = original.getWidth();
final int h = original.getHeight();
final BufferedImage cropped = w > h * aspectRatio ?
    original.getSubImage((w - h * aspectRatio) / 2, 0, h * aspectRatio, h) : 
    original.getSubImage(0, (h - w / aspectRatio) / 2, w, w / aspectRatio);
final Image tmp = cropped.getScaledInstance(size.width(), size.height(), Image.SCALE_SMOOTH);
```

Reply:
plus missing int casts here and there probably  (:

Reply:
Both work for me, altough centering would probably be easier on the eyes 😎

Reply:
made a new ticket about that: https://streamr.atlassian.net/browse/CORE-1265 just in case that becomes interesting in the future.